### PR TITLE
Project JS localStorage Example

### PR DESCRIPTION
**Pull request by Danny-Driscoll**
Creating a new folder for Project JS examples, and adding a first one that I am working on related to ITP follow-up.

### CODE REVIEW

File: Project-JS/clear_localstorage.js [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+    for (var key in window.localStorage){
+        if(key.indexOf('optimizely') !== -1){
+            window.localStorage.removeItem(key)
+        }
+    }
+}
+//The function can be called from Optimizely Project JS based on an available boolean flag, such as a cookie's value 
+var should_clear = window.customer_storage_flag_boolean;
+if(should_clear){
+    clearOptimizelyLocalStorage();
```
Comment (asaschachar) [inline Project-JS/clear_localstorage.js]: Looks like this should be named `clearOptlyLocalStorage` if you are trying to use the function above.

File: Project-JS/clear_localstorage.js [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+*/
+window.clearOptlyLocalStorage = function(){
+    for (var key in window.localStorage){
+        if(key.indexOf('optimizely') !== -1){
+            window.localStorage.removeItem(key)
+        }
+    }
+}
+//The function can be called from Optimizely Project JS based on an available boolean flag, such as a cookie's value 
+var should_clear = window.customer_storage_flag_boolean;
```
Comment (asaschachar) [inline Project-JS/clear_localstorage.js]: Should we name this variable something Optimizely specific? So it's less likely to collide with other variables?

**Comment by asaschachar** [comment]
I would also recommend adding a readme.md file next to the project JS, which explains how this is intended to be used and why.

Lastly, any thoughts on whether you would expect this project JS to be in the github library itself vs linked to from the library?

**Comment by Danny-Driscoll** [comment]
@asaschachar I think we need a first-party place to hold reference/approved Project JS implementations similar to the way we are able to do extensions for now. In this case, because it's one we specifically want to link to from the KB. I could also see us linking off to personal or third-party repos for future non-official recommendations or use cases.

**Comment by Danny-Driscoll** [comment]
@asaschachar Updated with individual Readme

**Pull request merged**